Npm install failed on postman executor: permissions missing

**Describe the bug**
I use a pre run script to install newman reporter but it failed due to permissions

```
Execution completed 
+ npm install
npm ERR! code EACCES
npm ERR! 
syscall mkdir
npm
 ERR! 
path /.npm
npm
 ERR! errno EACCES
npm ERR! 
npm ERR! Your cache folder contains root-owned files, due to a bug in
npm ERR!
 previous versions of npm which has since been addressed.
npm
 ERR! 
npm ERR!
 To permanently fix this problem, please run:
npm
 
ERR!
   sudo chown -R 1001:0 "/.npm" 
```

could we have a changement of owner to nmp to have the feature of npm install

**To Reproduce**
Steps to reproduce the behavior:
1. have a pre script with
```
#!/bin/sh

echo "Install html report ...."
cd /data/repo
npm install -g newman-reporter-html
npm install -g newman-reporter-htmlextra
```
 
2. run postman tests with executor args `--reporters cli,json,htmlextra` 

**Expected behavior**
have newman htmlreport

**Version / Cluster**
- 1.15

--- a/contrib/executor/postman/build/agent/Dockerfile
+++ b/contrib/executor/postman/build/agent/Dockerfile
@@ -2,6 +2,14 @@
 FROM postman/newman
 COPY postman /bin/runner
 RUN apk --no-cache add ca-certificates git
+
+# support npm reports
+RUN mkdir -p /.npm
+RUN chmod 777 /.npm
+RUN chmod 777 /root/
+RUN chmod 777 /usr/local/lib/node_modules
+RUN chmod 777 /usr/local/bin
+
 WORKDIR /root/
 EXPOSE 8082
 

--- a/contrib/executor/postman/build/agent/Dockerfile
+++ b/contrib/executor/postman/build/agent/Dockerfile
@@ -3,7 +3,7 @@ FROM postman/newman
 COPY postman /bin/runner
 RUN apk --no-cache add ca-certificates git
 
-# support npm reports
+# support npm report permissions
 RUN mkdir -p /.npm
 RUN chmod 777 /.npm
 RUN chmod 777 /root/

